File: problems/algorithms/function-fibonacci-sequence/js/response/fibonacci-sequence.js
function fibonacci(number) {
  let a = 0;
  let b = 1;
  let result = `${a}, ${b}`;
  let temp;

  if (number > 1)
    while (number > 2) {
      temp = a + b;
      a = b;
      b = temp;

      number--;
      result += `, ${temp}`;
    }
  else if (number == 1) return '0';
  else return '';

  return result;
}

export { fibonacci };

File: problems/algorithms/function-fibonacci-sequence/ruby/response/src/fibonacci_sequence.rb
def fibonacci(number)
  a = 0
  b = 1
  result = "#{a}, #{b}"
  temp = nil
  if number > 1
    while number > 2 do
      temp = a + b
      a = b
      b = temp
      number -= 1
      result += ", #{temp}"
    end
  elsif number == 1 
    return '0' 
  else return '' 
  end
  return result
end

